src/index.css

```diff
@@ -0,0 +1,23 @@
+* {
+  list-style: none;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family:
+    "Gill Sans",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
```

Comment:
Great job @Rafiduzzaman 🤝

- [ ] As stated in the project requirements, kindly ensure your design matches the [design provided](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) as much as possible. The text colors should be exact, the background colors should be more exact than your design. Take a closer look at the spacing between the cards of your design. Kindly improve your styles so that your design can replicate the design given.
- Your design
![Your design](https://github.com/Rafiduzzaman/react-capstone/assets/63917725/f1a576a8-6046-4346-bd87-476ccf15d0d4)
- Given design
![Given-design](https://github.com/Rafiduzzaman/react-capstone/assets/63917725/bb958fe0-1b59-446d-9ded-2214cf48d2a0)